### ISSUE TITLE
Bountiful

### ISSUE THREAD

**Issue by xJon** [xJon/The-1.12.2-Pack#78]
**If your feature request is a mod suggestion:**
1. Has someone suggested this mod before: **no**
2. Why should the mod be added: The mod adds the Bounty Board, which is customizable. This feature makes villages more interesting & gameplay more immersive.

**Mod info & download page**
* Link to official page: https://www.curseforge.com/minecraft/mc-mods/bountiful

**Comment by StefenSharkey** [comment]
This mod was updated to 1.12.2 in late 2021.
https://www.curseforge.com/minecraft/mc-mods/bountiful/files/3570157

**Comment by xJon** [comment]
@StefenSharkey Yes! I am aware - they backported some nice features. Not sure how actually useful these would be in this modpack though.

**Comment by StefenSharkey** [comment]
It does seem like it would be more suited towards a jobs or RP type of modpack. I know I personally wouldn't use this mod, but I focus solely on the tech mods; so, I have a skewed opinion.
